doc/en/user/source/community/mbtiles/installing.rst

```diff
@@ -9,4 +9,4 @@ Installing the GeoServer MBTiles extension
     #. Download the ``mbtiles-store-plugin`` from :download_community:`mbtiles-store` if you simply want to read MBTiles files. 
     #. Download the ``mbtiles-plugin`` from :download_community:`mbtiles` if you also want to use the WMS output format generaring MBTiles and the WPS process doing the same. Make sure to install corresponding WPS extension for GeoServer instance before installing this plugin, or GeoServer won't start.
 
-#. Extract the contents of the archive into the ``WEB-INF/lib`` directory of the GeoServer installation.
+#. Extract the contents of the archive into the ``GeoServer/webapps/geoserver/WEB-INF/lib`` directory of the GeoServer installation.
```

Comment:
This change makes sense for a bin package or a Windows installer, would not be the same path if GeoServer is installed somewhere else, e.g. in a Tomcat installation it would look something like ``apache-tomcat-9.0.55/webapps/geoserver/WEB-INF/lib`` (some intallations also rename the geoserver directory to something else, e.g, https://geoservice.dlr.de/eoc/ogc/features is a GeoServer, but the application has been renamed to "eoc").